modules/afsql/afsql.c

```diff
@@ -557,7 +557,7 @@ _table_create(AFSqlDestDriver *self, const gchar *table)
       return FALSE;
     }
 
-  g_string_printf(query_string, "CREATE TABLE %s (", table);
+  g_string_printf(query_string, "CREATE TABLE `%s` (`time` timestamp NOT NULL DEFAULT current_timestamp(), ", table);
```

Comment:
I think this could be added via configuration. You can specify default values to a field.